test/topojson/test.ml

```diff
@@ -98,24 +109,28 @@ let ezjsonm =
     (fun ppf t -> Fmt.pf ppf "%s" (Ezjsonm.value_to_string t))
     ezjsonm_equal
 
+(* type pp_set_formatter_out_functions = {
+     foreign_member : Topojson.Geometry.foreign_members -> unit;
+
+   } *)
 let main () =
   let s = read_file "./test_cases/files/exemplar.json" in
   let json = Ezjsonm.value_from_string s in
+
   let pp_ezjsonm ppf json = Fmt.pf ppf "%s" (Ezjsonm.value_to_string json) in
-  let pp_foreign_member ppf (v : (string * Ezjsonm.value) list) =
```

Comment:
This was correct before. Foreign members have the type `(string * Ezjsonm.value) list`

Reply:
Does that mean that `pp_ezjson` is also correct as it is?

Reply:
> let pp_ezjsonm ppf json = Fmt.pf ppf "%s" (Ezjsonm.value_to_string json) in

That looks good to me yep